.github/workflows/package-workflow.yml

```
@@ -25,13 +27,39 @@ jobs:
         version: 12.x
 
     - name: 'Build the standard bundle'
-      run: |
-        set -ex
-        node ./scripts/run-yarn.js build:cli
+      run: node ./scripts/run-yarn.js build:cli
 
     - name: 'Build the dist directory'
-      run: |
-        ./scripts/build-dist.sh
+      run: ./scripts/build-dist.sh
 
     - name: 'Build the deb archive'
       uses: ./scripts/actions/build-deb
+
+  windows-installer:
+    name: 'Windows Installer'
+    runs-on: windows-2016
+
+    steps:
+    - uses: actions/checkout@master
+
+    - uses: actions/setup-node@master
+      with:
+        version: 12.x
+
+    - name: 'Build the standard bundle'
+      run: node ./scripts/run-yarn.js build:cli
+
+    - name: 'Build the dist directory'
+      run: bash ./scripts/build-dist.sh
+
+    - name: 'Setup MSBuild'
+      uses: warrenbuckley/Setup-MSBuild@v1
```

Can we use a commit hash instead? I'd like to avoid referencing external actions through flexible references, as that could open an attack vector.
```suggestion
      uses: warrenbuckley/Setup-MSBuild#1176db82e64cd53b4944f0e57c2abec104b4782d
```

---

Hi

---

Good

---

Good idea! Will do.